utility/importer/brave_importer.cc

```diff
@@ -585,40 +587,48 @@ bool ParsePinnedSites(BraveLedger& ledger,
   return true;
 }
 
-void BraveImporter::ImportLedger() {
+bool BraveImporter::ImportLedger() {
   std::unique_ptr<base::Value> session_store_json = ParseBraveStateFile(
       "session-store-1");
   std::unique_ptr<base::Value> ledger_state_json = ParseBraveStateFile(
       "ledger-state.json");
-  if (!(session_store_json && ledger_state_json))
-    return;
+  if (!(session_store_json && ledger_state_json)) {
+    return false;
+  }
 
   BraveLedger ledger;
 
+  if (!ParsePaymentsPreferences(ledger, *session_store_json)) {
+    LOG(ERROR) << "Failed to parse preferences for Brave Payments";
+    return false;
+  }
+
   // It should be considered fatal if an error occurs while
   // parsing any of the below expected fields. This could
   // indicate a corrupt session-store-1
   if (!ParseWalletPassphrase(ledger, *session_store_json)) {
     LOG(ERROR) << "Failed to parse wallet passphrase";
-    return;
+    return false;
   }
 
-  if (!ParsePaymentsPreferences(ledger, *session_store_json)) {
-    LOG(ERROR) << "Failed to parse preferences for Brave Payments";
-    return;
-  }
+  if (ledger.settings.payments.enabled) {
```

Comment:
I prefer check `!ledger.settings.payments.enabled` and return false and put return true to the last

Reply:
Fixed! 😄